ssh/tailssh/tailssh.go

```diff
@@ -357,6 +361,24 @@ func (srv *server) handleAcceptedSSH(ctx context.Context, s ssh.Session, ci *ssh
 	// See https://github.com/tailscale/tailscale/issues/4146
 	s.DisablePTYEmulation()
 
+	var rec *recording // or nil if disabled
+	var err error
+	ptyReq, _, isPtyReq := s.Pty()
+	if recordSSH && isPtyReq { // for now only record pty sessions
```

Comment:
Can you move all the recording setup to a different function. Maybe have it return a ReadWriteCloser?

Reply:
Read?

And it needs to make two writer streams below. So, WriteCloser isn't enough.

There's definitely refactoring I want to do, but not necessarily to this part.